admin/tracking/class-tracking.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * @package WPSEO\Admin\Tracking
+ */
+
+/**
+ * This class handles the tracking routine.
+ */
+class WPSEO_Tracking implements WPSEO_WordPress_Integration {
+
+	/** @var string */
+	protected $transient_name = 'wpseo_tracking_last_request';
+
+	/** @var int */
+	protected $threshold = 0;
+
+	/** @var string */
+	protected $endpoint = 'https://search-yoast-poc-gdaxpa7udbwtvpgxqaufa3dejm.eu-central-1.es.amazonaws.com/yoast/tracking';
+
+	/**
+	 * Constructor setting the treshhold..
+	 */
+	public function __construct() {
+		$this->threshold = ( WEEK_IN_SECONDS * 2 );
+	}
+
+	/**
+	 * Registers all hooks to WordPress
+	 */
+	public function register_hooks() {
```

Comment:
This is not registering any hooks, just sending the tracking data.
Please rename method accordingly.